src/teleop/gui/src/store/index.js

```diff
@@ -0,0 +1,19 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import autonomy from './modules/autonomy'
+import controls from './modules/controls'
+import erd from './modules/erd'
+
+Vue.use(Vuex)
+
+const debug = process.env.NODE_ENV !== 'production'
+
+export default new Vuex.Store({
+  modules: {
```

Comment:
All of these modules and their associated files are again things I'm not entirely sure are necessary for this PR. If we merge them in now, I think there should be a ticket opened for cleaning up some of how these work later